ikabot/function/sellResources.py

```diff
@@ -76,159 +114,193 @@ def sellToOffers(s, city_to_buy_from, resource, e):
 		profit += amount * price
 
 	if len(chosen_offers) == 0:
-		e.set()
+		event.set()
 		return
 
-	available = city_to_buy_from['recursos'][resource]
+	available = city_to_buy_from['recursos'][resource_type]
 	amount_to_sell = min(available, total_amount)
 
 	banner()
 	print(_('\nHow much do you want to sell? [max = {}]').format(addDot(amount_to_sell)))
 	amount_to_sell = read(min=0, max=amount_to_sell)
 	if amount_to_sell == 0:
-		e.set()
+		event.set()
 		return
 
 	left_to_sell = amount_to_sell
 	profit = 0
 	for offer in chosen_offers:
-		cityname, username, amount, price, dist, destinyId = offer
+		cityname, username, amount, price, dist, destination_city_id = offer
 		cityname = cityname.strip()
 		amount = amount.replace(',', '').replace('.', '')
 		amount = int(amount)
 		price = int(price)
 		sell = min(amount, left_to_sell)
 		left_to_sell -= sell
 		profit += sell * price
-	print(_('\nSell {} of {} for a total of {}? [Y/n]').format(addDot(amount_to_sell), materials_names[resource], addDot(profit)))
+	print(_('\nSell {} of {} for a total of {}? [Y/n]').format(addDot(amount_to_sell), materials_names[resource_type], addDot(profit)))
 	rta = read(values=['y', 'Y', 'n', 'N', ''])
 	if rta.lower() == 'n':
-		e.set()
+		event.set()
 		return
 
-	set_child_mode(s)
-	e.set()
+	set_child_mode(session)
+	event.set()
 
-	info = _('\nI sell {} of {} in {}\n').format(addDot(amount_to_sell), materials_names[resource], city_to_buy_from['name'])
-	setInfoSignal(s, info)
+	info = _('\nI sell {} of {} in {}\n').format(addDot(amount_to_sell), materials_names[resource_type], city_to_buy_from['name'])
+	setInfoSignal(session, info)
 	try:
-		do_it1(s, amount_to_sell,  chosen_offers, resource, city_to_buy_from)
+		do_it1(session, amount_to_sell, chosen_offers, resource_type, city_to_buy_from)
 	except:
 		msg = _('Error in:\n{}\nCause:\n{}').format(info, traceback.format_exc())
-		sendToBot(s, msg)
+		sendToBot(session, msg)
 	finally:
-		s.logout()
-
-def createOffer(s, city, resource, e):
+		session.logout()
+
+def createOffer(session, city, resource_type, event):
+	"""
+	Parameters
+	----------
+	session : ikabot.web.session.Session
+	city : dict
```

Comment:
which city? offering or receiver?

Reply:
offering, creates an offer so that other players can buy from this city